build/dependencies.props

```diff
@@ -5,7 +5,9 @@
     <CoreFxVersion>4.4.0-*</CoreFxVersion>
     <DotnetDebToolVersion>2.0.0-preview2-*</DotnetDebToolVersion>
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
+    <JsonNetInMSBuildVersion>9.0.1</JsonNetInMSBuildVersion>
     <MoqVersion>4.7.49</MoqVersion>
+    <MsBuildPackageVersions>15.1.1012</MsBuildPackageVersions>
```

Comment:
Btw, seeing this makes me think it would be better if KoreBuild set common properties and package references so that each RepoTasks.csproj didn't need it....